presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -197,21 +196,33 @@ private boolean canAccessCatalog(Identity identity, String catalogName)
     @Override
     public void checkCanCreateSchema(Identity identity, CatalogSchemaName schema)
     {
+        if (!canAccessCatalog(identity, schema.getCatalogName())) {
```

Comment:
It is access control manager contract that ensures that these methods should not be called if user does not have access to catalog.

They don't have to be implemented. Revert.